packages/stats-py/tests/test_tier3.py

```
@@ -127,6 +127,8 @@ def test_cb_sem_semopy(df):
     source, target, estimate, std, se, z, p = structural["rows"][0]
     assert (source, target) == ("X1", "Y")
     assert estimate is not None and std is not None
+    # Structural path is estimated (not a fixed reference loading) → SE and z present too.
+    assert se is not None and z is not None
```

Ruff (0.15.20)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)